tests/contrib/pyramid/test_pyramid_autopatch.py

```diff
@@ -47,4 +47,4 @@ def test_config_include():
     """ This test makes sure that relative imports still work when the
```

Comment:
I don't want to be a mood killer, but this test talks about relative import. I don't know why it exists, but I feel like this change break what the author wanted to test.

Reply:
Good point! I'll investigate this further.